docs/preparing_datasets.md

```diff
@@ -0,0 +1,20 @@
+Preparing Datasets
+==================
+
+## Mortgage Example
+1. Setup [Apache Toree Jupyter notebook](/docs/toree.md).
+2. Download raw data from: https://rapidsai.github.io/demos/datasets/mortgage-data
+3. Run [Mortgage ETL job](https://github.com/rapidsai/spark-examples/blob/master/notebook/ETL/MortgageETL.ipynb).
```

Comment:
Link to file rather than master branch specific URL (?)
/notebook/ETL/MortgageETL.ipynb

Reply:
good eye